applications/app/controllers/SignupPageController.scala

```diff
@@ -24,25 +30,82 @@ class SignupPageController(
 
   val defaultCacheDuration: Duration = 15.minutes
+  val remoteRenderer = DotcomRenderingService()
 
-  def renderNewslettersPage(): Action[AnyContent] =
+  private def localRenderNewslettersPage()(implicit
+      request: RequestHeader,
+  ): Result = {
+    val groupedNewsletters: Either[String, GroupedNewslettersResponse] =
+      newsletterSignupAgent.getGroupedNewsletters()
+    groupedNewsletters match {
+      case Right(groupedNewsletters) =>
+        Cached(defaultCacheDuration)(
+          RevalidatableResult.Ok(
+            NewsletterHtmlPage.html(StaticPages.simpleNewslettersPage(request.path, groupedNewsletters)),
+          ),
+        )
+      case Left(e) =>
+        log.error(s"API call to get newsletters failed: $e")
+        NoCache(InternalServerError)
+    }
+  }
+
+  private def remoteRenderNewslettersPage()(implicit
+      request: RequestHeader,
+  ): Result = {
+
+    val newsletters: Either[String, List[NewsletterResponse]] =
+      newsletterSignupAgent.getNewsletters()
+
+    newsletters match {
+      case Right(newsletters) =>
+        Await.result(
+          remoteRenderer.getEmailNewsletters(
+            ws = wsClient,
+            newsletters = newsletters,
+            page = StaticPages.dcrSimpleNewsletterPage(request.path),
+          ),
+          3.seconds,
+        )
+      case Left(e) =>
+        log.error(s"API call to get newsletters failed: $e")
+        NoCache(InternalServerError)
+    }
+  }
+
+  def renderNewslettersPage()(implicit
+      executionContext: ExecutionContext = this.executionContext,
+  ): Action[AnyContent] =
+    csrfAddToken {
+      Action { implicit request =>
+        if (request.forceDCR) {
+          remoteRenderNewslettersPage()
+        } else {
+          localRenderNewslettersPage()
+        }
+      }
+    }
+
+  def renderNewslettersJson()(implicit
+      executionContext: ExecutionContext = this.executionContext,
+  ): Action[AnyContent] =
     csrfAddToken {
       Action { implicit request =>
-        val groupedNewsletters: Either[String, GroupedNewslettersResponse] =
-          newsletterSignupAgent.getGroupedNewsletters()
-
-        groupedNewsletters match {
-          case Right(groupedNewsletters) =>
-            Cached(defaultCacheDuration)(
-              RevalidatableResult.Ok(
-                NewsletterHtmlPage.html(StaticPages.simpleNewslettersPage(request.path, groupedNewsletters)),
-              ),
-            )
+        val newsletters: Either[String, List[NewsletterResponse]] =
+          newsletterSignupAgent.getNewsletters()
+
+        newsletters match {
+          case Right(newsletters) => {
+            val page = StaticPages.dcrSimpleNewsletterPage(request.path)
+            val dataModel =
+              DotcomNewslettersPageRenderingDataModel.apply(page, newsletters, request)
+            val dataJson = DotcomNewslettersPageRenderingDataModel.toJson(dataModel)
+            common.renderJson(dataJson, page).as("application/json")
```

Comment:
An interesting note on this is that we've never had a `.json` endpoint return DCR optimised data without requiring .json?dcr before. However unlike other cases, these pages didn't support .json as others have (generally /some-page.json returns the HTML in a .json with the html & the config object).

I'm not sure whether we should continue this restriction of requiring ?dcr or not. I will ask around and see if it might be a problem!

Reply:
Update: advice is to keep the .json?dcr pattern, so I guess 404 on `.json` & return the DCR json on `.json?dcr` :)